components/dashboard/src/settings/Plans.tsx

```diff
@@ -4,13 +4,461 @@
+        planCards.push(<PlanCard plan={applyCoupons(studentUnleashedPlan, appliedCoupons)} isCurrent={true} bottomLabel={bottomLabel}>{unleashedFeatures}</PlanCard>);
+    } else if (currentPlan.chargebeeId === unleashedPlan.chargebeeId) {
+        const bottomLabel = ('pendingSince' in currentPlan) ? <p className="text-green-600 animate-pulse">Upgrade in progress</p> : undefined;
+        planCards.push(<PlanCard plan={applyCoupons(unleashedPlan, appliedCoupons)} isCurrent={true} bottomLabel={bottomLabel}>{unleashedFeatures}</PlanCard>);
+    } else {
+        const targetPlan = applyCoupons(unleashedPlan, availableCoupons);
+        let onUpgrade;
+        switch (Plans.subscriptionChange(currentPlan.type, targetPlan.type)) {
+            case 'upgrade': onUpgrade = () => confirmUpgrade(targetPlan); break;
+        }
+        planCards.push(<PlanCard plan={targetPlan} isCurrent={false} onUpgrade={onUpgrade}>{unleashedFeatures}</PlanCard>);
+    }
+
     return <div>
         <SettingsPage title='Plans' subtitle='Manage account usage and billing.'>
-            <h3>Plans</h3>
+            <div className="w-full text-center">
+                <p className="text-xl font-medium text-gray-500">You are currently using the <span className="font-bold">{currentPlan.name}</span> plan.</p>
+                <p className="text-base">Upgrade your plan to get access to private repositories or more parallel workspaces.</p>
+                <p className="m-3 font-semibold text-gray-500">Remaining hours: {typeof(accountStatement?.remainingHours) === 'number'
+                    ? Math.floor(accountStatement.remainingHours * 10) / 10
+                    : accountStatement?.remainingHours}</p>
+                {(typeof(accountStatement?.remainingHours) === 'number' && typeof(currentPlan.hoursPerMonth) === 'number')
+                    ? <progress value={currentPlan.hoursPerMonth - accountStatement.remainingHours} max={currentPlan.hoursPerMonth} />
+                    : <progress value="0" max="100" />}
+                <p className="text-sm">
+                    <a className={`text-blue-light hover:underline" ${isChargebeeCustomer ? '' : 'invisible'}`} href="javascript:void(0)" onClick={() => { if (chargebeeClient) chargebeeClient.openPortal(); }}>Billing</a>
+                    {!!accountStatement && Plans.isFreePlan(currentPlan.chargebeeId) && <span className="pl-6">{currency === 'EUR'
+                        ? <>€ / <a className="text-blue-light hover:underline" href="javascript:void(0)" onClick={() => setCurrency('USD')}>$</a></>
+                        : <><a className="text-blue-light hover:underline" href="javascript:void(0)" onClick={() => setCurrency('EUR')}>€</a> / $</>}
+                    </span>}
+                </p>
+            </div>
+            <div className="mt-4 flex space-x-2">{planCards}</div>
```

Comment:
Wow, how big are your screens? 😅 On my 15" Macbook Pro, the current plan cards already barely fit inside my fullscreen window:

<img width="1440" alt="Screenshot 2021-03-30 at 18 23 39" src="https://user-images.githubusercontent.com/599268/113023136-9fa4fd00-9185-11eb-8c02-e5f5ad44e816.png">

This is what `space-x-7` and `w-56` looks like for me:

<img width="1440" alt="Screenshot 2021-03-30 at 18 26 04" src="https://user-images.githubusercontent.com/599268/113022979-7a17f380-9185-11eb-8925-b9fcee96fbe7.png">

Reply:
Oh, wait! The intent was to always fit the total width of the four cards to the content area on the right and not exceed the point of the avatar vertically. 🤯

Reply:
@jankeromnes An alternative approach here would be to switch to `w-44` and `space-x-3` on the parent container to keep the four columns here but some text would be truncated. To mitigate this we could repeat the plan benefits within the modal. WDYT? For this first iteration we could also skip repeating the benefits.

On smaller viewports we could have two rows with two cards and later on smaller viewports stacked cards but this is certainly  out of the scope here! 🤯

Reply:
@gtsiolis This is `w-44` and `space-x-3` -- better or worse? (Side note: I've also added tooltips to all features so that you can hover over them when they're truncated and still see the entire text.)

<img width="1440" alt="Screenshot 2021-03-31 at 10 25 28" src="https://user-images.githubusercontent.com/599268/113114309-7e86ef80-920b-11eb-9a9f-9a7ac990a91c.png">

Reply:
Personally, I think `w-52` was a better compromise (sure now it fits underneath the avatar, but I can't read most of the features anymore -- with `w-52` at least I could read almost all features). We can still iterate on responsive design later.

Reply:
For comparison, here is `w-52` with still `space-x-3` on the parent:

<img width="1440" alt="Screenshot 2021-03-31 at 10 31 16" src="https://user-images.githubusercontent.com/599268/113115051-4df38580-920c-11eb-9fc0-993fbda34971.png">

Reply:
`w-48` `space-x-3`:

<img width="1440" alt="Screenshot 2021-03-31 at 10 33 29" src="https://user-images.githubusercontent.com/599268/113115296-957a1180-920c-11eb-9c27-cd767bf5a6b3.png">

Reply:
@jankeromnes The issue here is that cards overflow the container. Let me illustrate this:

| Container |
|-|
| ![container](https://user-images.githubusercontent.com/120486/113115920-9a8f8e80-9215-11eb-91b7-f84ecbaec78b.png) |

Alternatively, or maybe better in this case, we could come up with a different design where plans are stacked vertically. Otherwise, we could leave this as is and change in the next milestone. WDYT?